thirdparty/pip/README.md

```diff
@@ -134,7 +134,7 @@ $(bazel info output_base)/external/pip<major>/
 
 Where:
 
-`<major>` is the major Python version for which the problem is occurring (indicated by `@pip2` or `@pip3`)
+`<major>` is the major Python version for which the problem is occurring (indicated by `@pip3` or `@pip3`)
```

Comment:
Probably would just remove or rewrite this sentence (and other references to `<major>` or similar)

Reply:
yeah I noticed all the copy-paste errors in this file while upgrading our internal deps... we have this same README in it's entirety there too!